lib/widget/button.dart

```diff
@@ -95,9 +99,12 @@ class ButtonState extends WidgetState<Button> {
     }
 
     // add margin if specified
-    return widget._controller.margin != null ?
-      Padding(padding: widget._controller.margin!, child: rtn) :
-      rtn;
+    return Container(
```

Comment:
Hmm, Button should configure its own width/height and not rely on the Container outside. Same goes with border and such. The only thing outside the button is probably the margin, and for that use a Margin widget.
If you want to change the width/height, see ThemeUtils.getButtonStyle(). Inside you you can set width/height, minWidth/minHeight, maxwidth/maxHeight.